lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import * as rtlUtils from '../../../__testutils__/rtlUtils';
+
+import userEvent from '@testing-library/user-event';
+
+import NxCombobox, {Props} from '../NxCombobox';
+
+describe('NxCombobox', function() {
+  const minimalProps: Props<string | number> = {
+        value: '',
+        onChange: () => {},
+        onSearch: () => {},
+        matches: []
+      },
+      rtlRender = rtlUtils.rtlRender(NxCombobox, minimalProps);
+
+  //mock scrollIntoView function using jest
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
```

Comment:
This should be done in the tests that need it, or if lots of tests need it, in a `beforeEach`. Not just directly within the `describe`.  Also `jest.spyOn(window.HTMLElement.prototype, 'scrollIntoView')` may be cleaner, though I'm not sure that works if the issue is that `scrollIntoView` just doesn't exist in `jsdom`

Reply:
I move it into the keyboard support `describe` block that uses the func, and also keep the old format since it doesn't exist in `jsdom`. Updated here 4029d58f1778f46cf53cdedad783b595e4a9da4e. Thanks.